pom.xml

```diff
@@ -540,6 +540,7 @@
         <module>server</module>
         <module>testing/base</module>
         <module>third-party</module>
+        <module>server/protocols/webadmin-cli</module>
```

Comment:
This module have to be added in server/pom.xml (and respect alphabetic ordering)